Exercises/operators-and-type-conversions/operators-and-type-conversions.js

```diff
@@ -0,0 +1,21 @@
+// Task #1
+a = 10;
+console.log(parseInt(a + '0'));
+
+// Task #2
+const stringValue = `Amazon annual revenue of US$232.887 billion`;
+let r = new RegExp(/\d+.\d+/);
+let annualRevenue = +stringValue.match(r);
```

Comment:
always `const` if no reassign

Reply:
const annualRevenue gives "TypeError: Assignment to constant variable" error.